KerbalStuff/blueprints/mods.py

```diff
@@ -557,12 +564,38 @@ def delete_version(mod_id: int, version_id: str) -> werkzeug.wrappers.Response:
         abort(404)
     if version[0].id == mod.default_version_id:
         abort(400)
+
+    protocol = _cfg('protocol')
+    cdn_domain = _cfg('cdn-domain')
+    if protocol and cdn_domain:
+        global _orig_create_connection
+        _orig_create_connection = connection.create_connection
+        connection.create_connection = create_connection_cdn_purge
+
+        requests.request('PURGE', protocol + '://' + cdn_domain + '/' + version[0].download_path)
+
+        connection.create_connection = _orig_create_connection
+
     db.delete(version[0])
     mod.versions = [v for v in mod.versions if v.id != int(version_id)]
     db.commit()
     return redirect(url_for("mods.mod", mod_id=mod.id, mod_name=mod.name, ga=game))
 
 
+def create_connection_cdn_purge(address: Tuple[str, Union[str, int, None]], *args: str, **kwargs: int) -> socket:
+    # Taken from https://stackoverflow.com/a/22614367
+    host, port = address
+
+    cdn_internal = _cfg('cdn-internal')
+    if cdn_internal:
```

Comment:
Let's make sure `host` is the same as `_cfg('cdn-domain')` before we override it.

Reply:
I had that check at first, but I realized that `cdn-domain` is actually `spacedock.info/content`.
So I think at most we could check `host in cdn_domain`.

Reply:
Or `cdn_domain.startswith(host)`?